pom.xml

```
@@ -25,6 +25,11 @@
         <java.version>11</java.version>
     </properties>
     <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-maven-plugin</artifactId>
+            <version>2.5.4</version>
```

You have multiple spring-boot dependencies now, you could add  `<spring-version>2.5.4</spring-version>` in the properties section, and reference it like 
```suggestion
            <version>${spring-version}</version>
```